hw/ip/rv_plic/data/rv_plic_bind_fpv.sv.tpl

```diff
@@ -38,8 +38,10 @@ module rv_plic_bind_fpv;
   bind rv_plic rv_plic_csr_assert_fpv rv_plic_csr_assert_fpv (
```

Comment:
It looks like the only reason why we need this template file here is the `${prio}` variable above. 

If we add
```hjson
    { name: "PrioWidth",
      desc: "Width of priority",
      type: "int",
      default: "${(prio).bit_length()}",
      local: "true",
    },
```
to https://github.com/lowRISC/opentitan/blob/9292494f0c54bf83cfa29f97cb85810877d7192f/hw/ip/rv_plic/data/rv_plic. hjson.tpl#L30

It should be possible to do without this template file as well. You just have to make sure that `.PRIOW` is set to `PrioWidth`.

Reply:
You may also want to propagate that parameter through the rest of the FPV testbench, as it seems that this is currently partially hardcoded.

Reply:
Thanks Michael for the clear suggestion. Let me try it and update this PR accordingly.